lib/BootBot.js

```diff
@@ -292,16 +292,20 @@ class BootBot extends EventEmitter {
 
     this._hearMap.forEach(hear => {
       if (typeof hear.keyword === 'string' && hear.keyword.toLowerCase() === text.toLowerCase()) {
-        captured = true;
-        return hear.callback.apply(this, [event, new Chat(this, senderId), {
-          keyword: hear.keyword
+        let res = hear.callback.apply(this, [event, new Chat(this, senderId), {
+          keyword: hear.keyword,
+          captured: captured
         }]);
-      } else if (hear.keyword instanceof RegExp && hear.keyword.test(text)) {
         captured = true;
-        return hear.callback.apply(this, [event, new Chat(this, senderId), {
+        return res;
+      } else if (hear.keyword instanceof RegExp && hear.keyword.test(text)) {
+        let res = hear.callback.apply(this, [event, new Chat(this, senderId), {
           keyword: hear.keyword,
-          match: text.match(hear.keyword)
+          match: text.match(hear.keyword),
+          captured: captured
```

Comment:
Use shorthand for implicit value here (simply `captured` without the `: captured` part).